python_bindings/tests/end-to-python-end-test/other_two_player_game_test.py

```diff
@@ -0,0 +1,636 @@
+|               ..@....|..|   |
+|               .......|...\ /
+|-1             --6----5....7..
+|               ........!../ \.
+|               ..........| ! |
+|
+|
+|-2
+|
+|
+"""
+
+
+def check_eighth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=-1, y=0),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.Bottom,
+        featureType=FeatureType.Road,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 4])
+
+    return game_id, game
+
+
+"""
+// Player1 places Straight road with city edge and places a meeple on the upper field.
+No one scores for the finished city
+|				  0	   1    2    3
+|
+|
+|               |   |.......!..
+|               .\ /...........
+|1              ..2....4-@--9--
+|               ./ \...|.../ \.
+|               |   |..|..|   |
+|          .....|   |..|..|   |
+|          ......\ /...|...\ /.
+|0         ..8----0----1....3..
+|          ..|.........!.../ \.
+|          ..@.........|..|   |
+|               ..@....|..|   |
+|               .......|...\ /
+|-1             --6----5....7..
+|               ........!../ \.
+|               ..........| ! |
+|
+|
+|-2
+|
+|
+"""
+
+
+def check_nineth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=2, y=1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.TopRightEdge,
+        featureType=FeatureType.Field,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 4])
+
+    return game_id, game
+
+
+"""
+// Player2 places Monastery with road, with a meeple on a monastery.
+Player2 scores 3 points for finished road
+|				  0	   1    2    3
+|
+|
+|               |   |.......!.......
+|               .\ /............[ ].
+|1              ..2....4----9---[A].
+|               ./ \...|.../ \..[@].
+|               |   |..|..|   |.....
+|          .....|   |..|..|   |
+|          ......\ /...|...\ /.
+|0         ..8----0----1....3..
+|          ..|.........!.../ \.
+|          ..@.........|..|   |
+|               ..@....|..|   |
+|               .......|...\ /
+|-1             --6----5....7..
+|               ........!../ \.
+|               ..........| ! |
+|
+|
+|-2
+|
+|
+"""
+
+
+def check_tenth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=3, y=1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.NoSide,
+        featureType=FeatureType.Monastery,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [4, 8])
+
+    return game_id, game
+
+
+"""
+// Player1 places T cross road, with a meeple on a bottom road
+player1 and player4 score 4 points for their roads
```

Comment:
a new contestant enters the game
```suggestion
player1 and player2 score 4 points for their roads
```

Reply:
fixed